code/modules/mob/living/carbon/human/human_attackalien.dm

```diff
@@ -30,7 +30,7 @@
 
 		if(M.a_intent == "disarm")
 			var/randn = rand(1, 100)
-			if (randn <= 80)
+			if (randn <= 100)	// Set to 100 to remove disarm/miss chance

```

Comment:
this breaks the code, it makes the other randn check below useless. Either remove those check or don't use 100.